src/react-recoil-hooks-testing-library.tsx

```diff
@@ -9,7 +9,7 @@ interface MockRecoilState {
 
 interface RenderHookOptions {
   states?: MockRecoilState[];
-  wrapper?: React.ComponentType;
+  wrapper?: React.ComponentType<any> | React.ComponentType;
```

Comment:
@spautz Looks like if we do this:

```suggestion
  wrapper?: React.FunctionComponent<any>;
```

We solve the issue of children not being inferred correctly.

Reply:
Great idea -- thanks for the suggestion